src/styles/index.scss

```diff
@@ -1,3 +1,46 @@
-body {
+@import './utils/variables';
+@import './utils/mixins';
+@import './blocks/header';
+@import './blocks/stars';
+@import './blocks/product-card';
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+body,
+li,
+ul,
+h1,
+h2,
+p {
   margin: 0;
+  padding: 0;
+}
+
+.main {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+}
```

Comment:
Consider renaming this class - it is better to avoid classes that have same name as HTML elements